frontend/views/utils/markdown-utils.js

```diff
@@ -193,3 +194,20 @@ export function combineMarkdownSegmentListIntoString (
     ''
   )
 }
+
+export function stripMarkdownSyntax (markdownString: string, truncateTo: number = -1): string {
+  markdownString = swapMentionIDForDisplayname(markdownString) // eg. '@identityContractID' -> '@user1'
+
+  const sanitized = markdownString
+    .replace(/\*\*(.*?)\*\*/g, '$1') // 'bold'
+    .replace(/_(.*?)_/g, '$1') // 'italic'
+    .replace(/~(.*?)~/g, '$1') // 'strike-through'
+    .replace(/`(.*?)`/g, '$1') // 'inline code'
+    .replace(/```/g, '') // 'code block'
```

Comment:
In this order, it produces this result (not fully stripped):

![Screenshot 2025-03-19 at 6 50 56 PM](https://github.com/user-attachments/assets/8c764a59-7094-4693-9624-dc3dbdd37b24)

Swapping these two (doing ``` first) produces:

![Screenshot 2025-03-19 at 6 51 56 PM](https://github.com/user-attachments/assets/6cfdf372-1959-4314-880e-16245144db50)

Which seems more correct to me since it actually removes the formatting, which is what we want a function called `stripMarkdownSyntax` to do.

Reply:
done it